hyperspectral/notebooks/spectra_demo.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demo of Spectrum and SpectralLibrary\n",
+    "\n",
+    "To run this demo, first download the complete ECOSTRESS spectral library data from the [ECOSTRESS website](https://speclib.jpl.nasa.gov/download) and unpack into a folder named `ECOSTRESS` in the current working directory.\n",
```

Comment:
If the user is expected to download an external package to `./ECOSTRESS`, consider `.gitignore`ing said directory.